src/containers/shared/components/Transaction/AMM/AMMDeposit/parser.tsx

```diff
@@ -0,0 +1,18 @@
+import { getAMMAccountID, findAssetAmount } from '../../../../MetaParser'
+import { XRP_BASE } from '../../../../transactionUtils'
+
+export function parser(tx: any, meta: any) {
+  const amount = tx.Asset
+  const amount2 = tx.Asset2
+  const accountID = getAMMAccountID(meta)
+  const fee = tx.Fee / XRP_BASE
+  amount2.amount = findAssetAmount(meta, amount2)
```

Comment:
Why does the amount need to come from the meta data?

Reply:
At the time of coding the amounts didnt exist in the tx body (only in the metadata) - but checking now it looks like its been updated to include the amounts and the assets. 

Getting the amounts from the metadata works well. Let me know if you'd like me to change it.